### ISSUE TITLE
Test fails on JRuby 9.2.0.0 - "sends a UTF-8 body"

### ISSUE THREAD

**Issue by olleolleolle** [cheald/manticore#75]
This test fails on JRuby 9.2.0.0.

```ruby
    it "sends a UTF-8 body" do
      response = client.post(local_server, body: "This is a post body ∑")
      expect(JSON.load(response.body)["body"]).to eq "This is a post body ∑"
    end
```

[Source location](https://github.com/cheald/manticore/blob/master/spec/manticore/client_spec.rb#L459-L462)

# Example failure:

```
  1) Manticore::Client#post sends a UTF-8 body
     Failure/Error: expect(JSON.load(response.body)["body"]).to eq "This is a post body ∑"
     
       expected: "This is a post body ∑"
            got: "This is a post body â\u0088\u0091"
     
       (compared using ==)
     # ./spec/manticore/client_spec.rb:461:in `block in (root)'
```

Test _passes_ on 9.0.5.0 - https://travis-ci.org/cheald/manticore/jobs/383309866

Build 221 overview - https://travis-ci.org/cheald/manticore/builds/383309863

**Comment by olleolleolle** [comment]
OH: "Smells JI"

**Comment by cheald** [comment]
The fix for #70 and treating our test JSON payloads as application/json should fix this.

**Comment by olleolleolle** [comment]
@cheald Huzzah!
